server/models/Collaborator.js

```diff
@@ -0,0 +1,11 @@
+import mongoose from 'mongoose';
+
+const { Schema } = mongoose;
+
+const collaboratorSchema = new Schema({
+  toDoId: String,
```

Comment:
Is there any reason you decided your toDoId should be a String as opposed to Integer 🤔 ?

Reply:
The toDoId is a MongoDB ObjectId. Here is an example: 59f30f493cd15b2d246f28db

Reply:
Ok